Common/Orders/KrakenOrderProperties.cs

```diff
@@ -0,0 +1,35 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+
+using System;
+
+namespace QuantConnect.Orders
+{
+    public class KrakenOrderProperties : OrderProperties
+    {
+        /// <summary>
+        /// Comma delimited list of order flags. viqc = volume in quote currency (not currently available), fcib = prefer fee in base currency, fciq = prefer fee in quote currency,
+        /// nompp = no market price protection, post = post only order (available when ordertype = limit)
+        /// </summary>
+        public string Oflags { get; set; }
```

Comment:
I think we should split out the different order settings as their own properties the users can enable/disable/set, `Comma delimited list of order flags` is going to get error prone and messy. Suggest reviewing existing order properties for other brokerages

Reply:
`Oflags` can now be removed?

nit - Same for `using System;` really